GPU/GLES/GLES_GPU.cpp

```diff
@@ -458,6 +458,11 @@ GLES_GPU::GLES_GPU(GraphicsContext *ctx)
 	glstate.Restore();
 	transformDraw_.RestoreVAO();
 	textureCache_.NotifyConfigChanged();
+
+	// Load shader cache.
+	File::CreateFullPath(GetSysDirectory(DIRECTORY_CACHE));
+	shaderCachePath_ = GetSysDirectory(DIRECTORY_CACHE) + "/" + g_paramSFO.GetValueString("DISC_ID") + ".shadercache";
+	shaderManager_->LoadAndPrecompile(shaderCachePath_);
```

Comment:
I think, on Android, we should ideally save in this directory instead:

```c++
void HandleGlobalMessage(const std::string &msg, const std::string &value) {
	if (msg == "inputDeviceConnected") {
		KeyMap::NotifyPadConnected(value);
	}

	if (msg == "cacheDir") {
		DiskCachingFileLoaderCache::SetCacheDir(value);
	}
}
```

Probably should propagate the variable better somewhere else.  The benefits of this directory:

1. If the user downloads the PSP/ folder from a device (let's say a Mali-400) to their PC, it won't bring along the tainted device-specific shader cache (unless I'm wrong and it's not tainted?)

2. If the files are somehow big (seems unlikely), they seem to get auto-deleted by the OS.  This might matter more if we tried to shove shader binaries in there, though.

3. It provides an easy path for the user to clear the cache, in case they are having problems.

-[Unknown]